tutorxqueue/plugin.py

```diff
@@ -28,25 +29,33 @@
     },
 }
 
-templates = pkg_resources.resource_filename("tutorxqueue", "templates")
-hooks = {
-    "init": ["mysql", "xqueue"],
-    "build-image": {"xqueue": "{{ XQUEUE_DOCKER_IMAGE }}"},
-    "remote-image": {"xqueue": "{{ XQUEUE_DOCKER_IMAGE }}"},
-}
-
-
-def patches():
-    all_patches = {}
-    for path in glob(
-        os.path.join(pkg_resources.resource_filename("tutorxqueue", "patches"), "*")
-    ):
-        with open(path) as patch_file:
-            name = os.path.basename(path)
-            content = patch_file.read()
-            all_patches[name] = content
-    return all_patches
-
+# Inizialization hooks
+tutor_hooks.Filters.COMMANDS_INIT.add_item((
+    "mysql",
+    ("xqueue", "tasks", "mysql", "init"),
+))
+
+tutor_hooks.Filters.COMMANDS_INIT.add_item((
+    "xqueue",
+    ("xqueue", "tasks", "xqueue", "init"),
+))
+
+# Image managment
+tutor_hooks.Filters.IMAGES_BUILD.add_item((
+    "xqueue",
+    ("plugins", "xqueue", "build", "xqueue"),
+    "{{ XQUEUE_DOCKER_IMAGE }}",
+    (),
+))
+
+tutor_hooks.Filters.IMAGES_PULL.add_item((
+    "xqueue",
+    "{{ XQUEUE_DOCKER_IMAGE }}",
+))
+tutor_hooks.Filters.IMAGES_PUSH.add_item((
+    "xqueue",
+    "{{ XQUEUE_DOCKER_IMAGE }}",
+))
 
 @click.group(help="Interact with the Xqueue server")
 def command():
```

Comment:
In order for the command to be available from the CLI you will have to implement the CLI_COMMANDS filter https://docs.tutor.overhang.io/reference/api/hooks/consts.html#tutor.hooks.Filters.CLI_COMMANDS
I guess this should be part of the cookiecutter conversion tutorial... Please feel free to open a PR there as well.

Reply:
I mean: the filter already exists. All you have to do is to actually use it. Something like:

```
tutor_hooks.Filters.CLI_COMMANDS.add_item(command)
```

Reply:
Thank you @regisb !!I already added the filter and verify that the commands are working properly. Could you please check it again!!

Reply:
According to my tests, the `tutor xqueue` command does not exist... but the `tutor command` command does! That's because of the function name. You can either rename the function or add a `name="xqueue"` argument to the `@click.group` call. 

Can you please check that following all the instructions from the README works in Nutmeg?